src/main/java/com/volmit/iris/core/CommandManager.java

```diff
@@ -19,7 +19,7 @@
 package com.volmit.iris.core;
 
 import com.volmit.iris.Iris;
-import com.volmit.iris.core.decrees.CMDIris;
+import com.volmit.iris.core.decrees.DecIris;
```

Comment:
Justify this

Reply:
Clarify the system is using Decree more than CMD
Also abbreviations shouldn't be all caps imo because of style.
Decree is our replacement for commands.

Reply:
tbh I would consider renaming them to DecreeIris<Sub> to match the current CommandIris<Sub>

Reply:
Once we actually transfer lets call it CommandIris again, and CommandStudio, because they ARE commands, but for the time being i think dec should be fine.

Reply:
👌